Enterprise-CMCS/macpro-mdct-mfp
# WP Admin Dashboard

### Description

Added admin dashboard for admin accounts. There are a couple of differences between state user & an admin.

Column
- Due Date (State user only)
- '#' (Admin only)
- Button Text = "View"
- Unlock (Admin only)
- Archive (Admin only)

When clicking unlock, a modal will popup informing the user they have unlocked the form.

**NOTES** 
- Unlock functionality currently doesn't fully work. Once we have submission and approval implemented, that would be a good time to add / fix it.
- The video shows forms with statuses: Approved & Submitted, I faked those by editing the dynamodb data for this branch in aws. They aren't real so click with caution.


https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/127151429/b68e34cf-3cb4-4e85-be53-c57a3053684d



### Related ticket(s)


MDCT-2993

---
### How to test

Use Deploy Link: https://d15xrd0qfab0yt.cloudfront.net/  

1) Sign into MFP state user, i.e. ([stateuser@test.com](mailto:stateuser@test.com)). this location is Puerto Rico.
2) Create a new WP form, doesn't matter the data.
3) Sign out
4) Sign into MFP admin (adminuser@test.com), select Puerto Rico
5) Select WP radio button.
6) You should see the admin dashboard showing the WP you had created earlier.
7) You can click around the buttons on the admin dashboard.

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
